build.cake

```diff
@@ -40,10 +40,12 @@ var srcProjectNames = new[]
 };
 
 Task("Default")
-    .IsDependentOn("TestAndPackage");
+    .IsDependentOn("Test");
 
-Task("TestAndPackage")
+Task("Release")
+    .IsDependentOn("Build")
```

Comment:
I'm surprised this line wasn't here already.

Reply:
It's somewhat redundant.

Test also depends on Build.